src/app-context/app-provider.js

```diff
@@ -12,6 +12,9 @@ const query = {
             fields: ['authorities', 'organisationUnits[id,path]'],
         },
     },
+    system: {
+        resource: 'system/info',
+    },
```

Comment:
I believe you don't need to do this. Use the `useConfig` hook from `app-runtime` instead. This will save us from doing an additional request too.

Reply:
I forgot `useConfig` contained system settings :sweat_smile: